.github/ISSUE_TEMPLATE/mismatch-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Mismatch report'
+about: Template for reporting a new entry for mismatch database
+title: 'feat: new product mismatch [product name]'
+labels: enhancement
```

Comment:
I think this is more of a bug than an enhancement (again, in the eyes of the user)
```suggestion
labels: bug
```